frontend/packages/console-plugin-sdk/src/codegen/__tests__/plugin-resolver.spec.ts

```diff
@@ -181,4 +207,20 @@ describe('plugin-resolver', () => {
       ]);
     });
   });
+
+  describe('getMonorepoRootDir', () => {
+    it('returns the location of Console monorepo root package', () => {
+      const currentPackageFile = findUp.sync('package.json', {
+        cwd: __dirname,
+      });
+      expect(fs.existsSync(currentPackageFile)).toBe(true);
+
+      const parentPackageFile = findUp.sync('package.json', {
+        cwd: path.join(path.dirname(currentPackageFile), '..'),
+      });
+      expect(fs.existsSync(parentPackageFile)).toBe(true);
+
+      expect(getMonorepoRootDir()).toEqual(path.dirname(parentPackageFile));
```

Comment:
Note: this test case relies on Console monorepo file structure, exercising the `getMonorepoRootDir` function from a different angle (compared to its implementation).

To make this test case "pure" we'd have to mock all the dependencies: `findUp.sync`, `readPkg.sync`, `fs.existsSync`. However, this would complicate the test code.

Let me know if you think we should take a different approach here.